src/driver-hidpp20.c

```diff
@@ -1423,7 +1423,7 @@ hidpp20drv_commit(struct ratbag_device *device)
 			continue;
 
 		rc = hidpp20drv_update_report_rate(profile, profile->hz);
-		if (rc)
+		if (rc && rc != -ENOTSUP)
```

Comment:
doesn't this mean we lose the information that the current report rate couldn't be set? e.g. if I want to change the rate it would now show as success rather than error? Do we need some extra check for whether it differs from the current rate or something?

Reply:
In #891 I added errors to the log to for this cases. Do we have a better way to deal with this? Well, actually I think we could have a dirty flag.